src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.scss

```diff
@@ -1,9 +1,13 @@
-.synced {
-    color: #3e8acc;
+.notSynced {
+    --fa-secondary-color: var(--bs-body-color);
+    --fa-secondary-opacity: 1; // necessary to set the opacity to 1, because it's 0.4 per default
+    --fa-primary-color: var(--warning);
 }
 
-.notSynced {
-    color: #e8bf48;
+.saved {
+    --fa-secondary-color: var(--bs-body-color);
+    --fa-secondary-opacity: 1;
+    --fa-primary-color: var(--success);
 }
```

Comment:
These two classes are duplicated from `src/main/webapp/app/exam/participate/exam-navigation-sidebar/exam-navigation-sidebar.component.scss` - could we move them to a shared file?

Reply:
I will reuse the scss file of navigation-sidebar-component